src/clients/SparkCore.js

```diff
@@ -733,9 +745,9 @@ class SparkCore extends EventEmitter {
     data: Buffer,
     ..._:void[]
   ): Promise<*> => {
-    /*TODO: data type! */
+    // TODO: data type!
```

Comment:
ok..ill check this first, then delete. also what these ` ..._:void[]`  unused arguments for, in some methods?